index.html

```diff
@@ -2758,10 +2758,21 @@ <h3>Terms of Use</h3>
 {
   "@context": [
     "https://www.w3.org/2018/credentials/v1",
-    "https://www.w3.org/2018/credentials/examples/v1"
+    "https://www.w3.org/2018/credentials/examples/v1",
+    {
+        "VerifiablePresentationExtension": {
+          "@id": "https://www.w3.org/2018/credentials#VerifiablePresentationExtension",
+          "@context": {
+            "termsOfUse": {
```

Comment:
Examples should always use `@protected`.

```suggestion
            "@protected": true,
            "termsOfUse": {
```